visualization/src/point_picking_event.cpp

```diff
@@ -167,49 +199,67 @@ pcl::visualization::PointPickingCallback::performSinglePick (
 /////////////////////////////////////////////////////////////////////////////////////////////
 int
 pcl::visualization::PointPickingCallback::performAreaPick (vtkRenderWindowInteractor *iren,
-                                                           std::vector<int> &indices)
+                                                           std::vector<std::vector<int> > &indices)
```

Comment:
Deprecate the original function, redirect it through the new call

Reply:
Is this a public API though? I'd say we don't need to go through deprecation ceremonies if this is not supposed to be utilized directly by the end user.

Reply:
The header isn't in impl, the namespace is main namespace, and PCL_EXPORTS doesn't do anything on GCC. On the surface, it fits the criteria of a public API

Reply:
Technically it's public, yes. But it is only used inside `InteractorStyle`, this is not an interface for end user. The end user is supposed to provide a function with signature

```cpp
void (const pcl::visualization::PointPickingEvent&)
```

to the the method `registerPointPickingCallback()` of the visualizer class.

Reply:
That's reasonable, I guess 👍